src/app/shared/components/create-teacher/teacher-form/teacher-form.component.scss

```diff
@@ -0,0 +1,91 @@
+.form{
+  &_label{
+    font-weight: 500;
+  }
+  &_required{
+    color:red;
+  }
+  &_textarea{
+    min-height: 200px;
+    border: 1px solid #E3E3E3;
+    box-sizing: border-box;
+    border-radius: 21px;
+    padding: 1.2rem;
+    margin-bottom: 1.2rem;
+  }
+  &_input{
+    border: 1px solid #E3E3E3;
+    box-sizing: border-box;
+    border-radius: 21px;
+    padding:0.5rem;
+    
+    &_date{
+      max-width: 30%;
+    }
+  }
+}
+.btn{
+  color:white;
+  background-color:#3849F9;
+  border: 1px solid #3849F9;
+  box-sizing: border-box;
+  border-radius: 20px;
+  padding: 2px 30px;
+  margin: 0 1rem;
+  &_reset{
+    color: #3849F9;
+  }
+}
+input:focus,
+textarea:focus,
+select:focus,
+button:focus {
+  outline: none;
+}
+::placeholder {
+  color:#333333;
+  font-family: Open Sans;
+  font-style: normal;
+  font-weight: normal;
+}
+  
+.div-input{
+  width: 95px;
+  height: 58px;
+  background: rgba(56, 73, 249, 0.1);
+  border: 1px dashed #3849F9;
+  box-sizing: border-box;
+  border-radius: 5px;
+  margin: 1rem;
+  
+&:hover{
+  cursor:pointer;
+  opacity: 0.5;
+  }
+}
+  
+.mat-icon-add {
+  position: relative;
+  top: 17px;
+  left: 35px;
+  vertical-align: middle;
+  height: 26px;
+  color: #3849F9;
+}
+.mat-icon-delete{
+  color: #3849F9;
+}
+input[type="file"] {
+  display: none;
+}
+  
+::ng-deep .mat-form-field-underline,
+::ng-deep .mat-form-field-label-wrapper {
+  display: none;
+}
+::ng-deep .mat-form-field-infix {
+  border:none;
+}
+.form_border{
```

Comment:
what css model do you use? I can see both variants dash and underscore delimiters in your code